packages/flutter/lib/src/material/slider.dart

```diff
@@ -757,6 +765,7 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
         max: widget.max,
         divisions: widget.divisions,
         activeColor: widget.activeColor,
+        thumbColor: widget.cupertinoThumbColor ?? CupertinoColors.white,
```

Comment:
What about dark mode?

Reply:
I can't find any reference about colors based on the app theme for this Widget, @clocksmith do you have any reference about this?